src/test/FuzzerImpl.cpp

```diff
@@ -430,17 +571,44 @@ getFuzzConfig(int instanceNumber)
 }
 
 static void
-resetRandomSeed()
+resetRandomSeed(uint seed)
 {
     // seed randomness
-    srand(1);
-    gRandomEngine.seed(1);
+    srand(seed);
+    gRandomEngine.seed(seed);
 }
 
-static void
-resetTxInternalState(Application& app)
+FuzzerImpl::FuzzerImpl()
 {
-    resetRandomSeed();
+    resetRandomSeed(1);
+
+    static_assert(NUM_RANDOM_LEDGER_KEYS < NUM_STORED_LEDGER_KEYS,
+                  "No room for non-random ledger keys");
+    auto const firstRandomLedgerKey =
+        NUM_STORED_LEDGER_KEYS - NUM_RANDOM_LEDGER_KEYS;
+
+    for (auto index = 0; index < firstRandomLedgerKey; ++index)
+    {
+        mPostInitLedgerKeys[index] =
+            LedgerEntryKey(LedgerTestUtils::generateValidLedgerEntry());
+    }
+
+    for (auto index = firstRandomLedgerKey; index < NUM_STORED_LEDGER_KEYS;
+         ++index)
+    {
+        size_t const entrySize = 3;
+        mPostInitLedgerKeys[index] =
```

Comment:
> Also if you want these to be invalid, there is no guarantee that this will do that. For example, the key for any ACCOUNT generated here will be valid.

True.  At first at least, I'm not thinking of a way of _guaranteeing_ that they're all invalid that feels worth the complexity to me -- maybe I should just rename these from "Invalid" to "Unvalidated" and not care if some of them turn out to be valid.  What do you think?

Reply:
(So far, in the latest push, that's all I've done -- rename "invalid" to "unvalidated".)